### ISSUE TITLE
Why using <table> as columns?

### ISSUE THREAD

**Issue by charlesmudy** [mailchimp/email-blueprints#25]
Hi,

Thanks for the open-source HTML email template, I am a bit confused as to why the blueprint template is using table as columns, for me - it just doesn't work and somehow doesn't make any sense in terms of natural structure for html table formatting.
![table-rows-columns-smaller](https://cloud.githubusercontent.com/assets/1445007/3159178/587d93f4-eb0d-11e3-9c34-d60a8c11c770.png)

I'm guessing I might be wrong, please can anyone provide any technical reason behind this, I have changed it for td and it works perfectly for me but I would still need to know.

Thanks in advance.
Shina

**Comment by bago** [comment]
check Outlook 2007+, Gmail and responsive behaviour of your table using td vs table using nested tables.

**Comment by charlesmudy** [comment]
It works perfectly fine, you can see it here https://github.com/charlesmudy/Responsive-HTML-Email-Template

Test it and let me know

**Comment by bago** [comment]
That template nest a lot of tables. You probably fixed outlook issues by nesting more tables.
In your template there are up to 6 nested tables while this templates uses at most 4 nested tables.
I remember many issues with older Lotus Notes when using many nested tables.

**Comment by charlesmudy** [comment]
Yes indeed there a bit more than the blue-print but the result is better which is the main goal and not only on outlook but also on in-browser emails like gmail and hotmail. You can check the demo on lotus, it works fine on 8+ if I remember correctly but usually, the open rates of lotus notes is very very low.

The nested tables are meant for proper blocks and each sections are divided by rows (<tr>). Remember that we want the contents to stay the same for platform that don’t support media queries, nesting the tables helps target it properly.

Play with it and let me know if you can come up with something simpler.

**Comment by bago** [comment]
I don't have a Lotus Notes 6 here to test the table nesting. BTW, I was answering to your question. Maybe I misunderstood your question.

You asked why the templates uses tables instead of td for columns.. My answer is "because using tds would need one more tag in the middle".

Maybe you should provide code snippets to show what's this issue about (I can't make sense of the screenshot you posted)

**Comment by TF-Ryan-LaMountain** [comment]
To test your email code on more obscure email clients like Lotus Notes 6, you might check out Email on Acid and Litmus.

**Comment by bago** [comment]
Sure, I know... I don't use them because they don't provide previews for most italian email providers.
I don't care at all about Lotus 6, I was just answering your question ;-)
